ScoutSuite/providers/gcp/rules/findings/kms-encryption-keys-not-rotated.json

```diff
@@ -35,8 +35,8 @@
             "or",
             [
                 "kms.projects.id.keyrings.id.keys.id.next_rotation_time_days",
-                "greaterThan",
-                "90"
+                "false",
```

Comment:
In terms of logic the previous implementation made more sense - you're no longer storing rotation information, only the result of the check.

Reply:
Went back to the previous implementation